catalog/plugin/catalog-plugin-videothumbnail/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+    <OCD name="Video Thumbnail Plugin"
+         id="org.codice.ddf.catalog.content.plugin.video.VideoThumbnailPlugin">
+        <AD description="Maximum video file size in Megabytes for which to create a thumbnail. Default is 120 Megabytes."
```

Comment:
I think it would be good to include in the description something about increasing the size could affect system performance.